ci/run_envoy_docker.sh

```diff
@@ -117,8 +117,6 @@ fi
 docker run --rm \
        "${ENVOY_DOCKER_OPTIONS[@]}" \
        "${VOLUMES[@]}" \
-       -e AZP_BRANCH \
```

Comment:
We depend on AZP_BRANCH, which is used to determine if we should skip some tests:

https://github.com/envoyproxy/nighthawk/blob/9e68bd31dfeaf71fd47f6921d4a6b2cd0b74a08f/ci/do_ci.sh#L116

https://github.com/envoyproxy/nighthawk/blob/9e68bd31dfeaf71fd47f6921d4a6b2cd0b74a08f/test/integration/utility.py#L113

https://github.com/envoyproxy/nighthawk/blob/9e68bd31dfeaf71fd47f6921d4a6b2cd0b74a08f/test/integration/test_integration_basics.py#L146

Reply:
See above response.